stage-ovos/00-system/files/ovos-admin-phal.service

```diff
@@ -11,4 +11,4 @@ User=root
 ExecStart=/usr/libexec/ovos-systemd-admin-phal
 
 [Install]
-WantedBy=ovos.service
+WantedBy=multi-user.target
```

Comment:
I think this should still be `ovos.service`.  without ovos, why have this

Reply:
It gets installed as multi-user.target. We need that hook. If you look three lines above there's a `PartOf` reference.